lib/miq_automation_engine/engine/miq_ae_event.rb

```diff
@@ -31,10 +33,13 @@ def self.raise_synthetic_event(event, inputs, message = nil)
       aevent      = build_evm_event(event, inputs)
     end
 
-    MiqAeEngine.deliver(obj_type, obj_id, aevent, instance, nil, nil, message)
+    user = automate_user
+    MiqAeEngine.deliver(obj_type, obj_id, aevent, instance, nil, user.id, message)
   end
 
   def self.raise_evm_event(event_name, target, inputs = {}, _message = nil)
+    user = automate_user
```

Comment:
@lfu yes, we want to change that this week. but for now, we just want to get the user flowing through the system, and after you do your research, we can get a pr fixing these event user ids.